xmsannotator/R/compute_rt_modules.R

```diff
@@ -95,19 +98,20 @@ compute_cluster_assignments <- function(clusters, data) {
 #'
 #' @param peak_table Feature table with columns ['rt', 'module','peak'].
 #' @param peak_width Estimated chromatographic peak width.
+#' @param show If TRUE, plot the resulting clustering. 
 #'
 #' @return
```

Comment:
Could you also add a short description of the returned table here?

Reply:
Good point, since this is one thing where we wanted to introduce some clarity!

Reply:
Added this in d072c8a38f9358446da9b6f5ee2a4356795e7658